src/modules/users/decorators/swagger.decorator.ts

```diff
@@ -0,0 +1,67 @@
+import { applyDecorators } from '@nestjs/common';
```

Comment:
create file for every decorator, not in one file,

Reply:
I also thought about it at first but then I decided it is better to have it in one place like we keep functionality of controller and service in one file. Also they all serve one purpose which is adding swagger documentation to users controller. Right now each decorator is very small and personally I think it is more readable and convenient to have them in one file.